src/index.ts

```diff
@@ -65,8 +65,6 @@ export { default as PageHeader } from './components/PageHeader';
 export { default as PageLevelBanner } from './components/PageLevelBanner';
 export { default as Panel } from './components/Panel';
 export { default as Popover } from './components/Popover';
-export { default as PopoverButton } from './components/PopoverButton';
```

Comment:
Nice, as subcomponents these shouldn't have been default exported in the first place